### ISSUE TITLE
UI Element for own access rights

### ISSUE THREAD

**Issue by ottmann** [Viva-con-Agua/drops#57]
Users should have an overview for theirs access rights. It should also be possible to have an overview from all users, which has the same access right.

**Comment by johannsell** [comment]
Good point. But a list of users holding the same access rights is a kind of awareness support. So, we have to think about the corresponding pipeline (e.g. Could you see all users or only users of your own crew? What about employees? ...)
